roles/reproducer/tasks/configure_crc.yml

```diff
@@ -5,6 +5,66 @@
   ansible.builtin.slurp:
     path: /etc/ci/env/networking-environment-definition.yml
 
+- name: Configure CRC services
+  delegate_to: crc-0
+  vars:
+    _parsed: "{{ _net_conf.content | b64decode | from_yaml}}"
+    _crc: "{{ _parsed.instances['crc-0'].networks.ctlplane }}"
+  block:
+    - name: Check which dnsmasq config we must edit
+      register: _dnsmasq
+      ansible.builtin.stat:
+        path: '/srv/dnsmasq.conf'
+        get_attributes: false
+        get_checksum: false
+        get_mime: false
+
+    - name: Configure local DNS for CRC pod
+      become: true
+      vars:
+        _config_file: >-
+          {{
+            _dnsmasq.stat.exists |
+            ternary(_dnsmasq.stat.path, '/etc/dnsmasq.d/crc-dnsmasq.conf')
+          }}
+      block:
+        - name: Configure local DNS for CRC pod
+          register: last_modification
+          ansible.builtin.replace:
+            path: "{{ _config_file }}"
+            regexp: "192.168.130.11"
+            replace: "{{ _crc.ip_v4 }}"
+
+        - name: Configure local DNS for CRC pod with ip_v6
+          when:
+            - _crc.ip_v6 is defined
+            - _crc.ip_v6 | length > 0
+          ansible.builtin.lineinfile:
+            path: "{{ _config_file }}"
+            line: "address=/{{ item  }}/ {{ _crc.ip_v6 }}"
```

Comment:
@hjensas heh, so here this is how CRC is configuring its DNS - all "address". We can either do like them, or "fix" the v6 version - meaning a list of key/var:
```yaml
loop:
  - key: apps-crc.testing
    type: address
  - key: api.crc.testing
    type: host-record
  - key: api-int.crc.testing
    type: host-record
```
but then we will face the syntax differences - `host-record=foo,IP` vs `address=/foo/IP`

dnsmasq is so easy -.-....

@cescgina FTR, "address" creates a wildcard (*.foo), while apparently we should get simple A/AAAA for api/api-int records...

IMHO we can stick with the current thing - but in any cases, I'd rather go with a `blockinfile` instead of a loop. Faster, and... well, there we can use the right type :).